.travis.yml

```diff
@@ -1,23 +1,11 @@
 sudo: false
-
 language: node_js
-
 node_js:
+  - '10'
+  - '8'
   - '6'
   - '4'
   - '0.12'
```

Comment:
When will abandoning 0.12 and 0.10 be on our radar?

Reply:
I'm planning to drop it in gulp 5